dev-libs/cjson/cjson-1.7.18.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Ultralightweight JSON parser in ANSI C"
+DESCRIPTION="cJSON aims to be the dumbest possible parser that you can get your job done with. \
+It's a single file of C, and a single header file.
+
+JSON is described best here: http://www.json.org/
+It's like XML, but fat-free. You use it to move data around, store things, or just generally \
+represent your program's state.
+
+As a library, cJSON exists to take away as much legwork as it can, but not get in your way. As a \
+point of pragmatism (i.e. ignoring the truth), I'm going to say that you can use it in one of \
+two modes: Auto and Manual. Let's have a quick run-through.
+
+I lifted some JSON from this page: http://www.json.org/fatfree.html That page inspired me to \
```

Comment:
Wouldn't it be handy if url's like these could be made `click-able` in the HaikuDepot window?